commands/team_users.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var TeamUsersCmd = &cobra.Command{
+	Use:   "users",
+	Short: "Management of team users",
+}
+
+var RemoveUsersCmd = &cobra.Command{
```

Comment:
To keep consistent with the naming schema, we should use:

```suggestion
var TeamUsersRemoveCmd = &cobra.Command{
```

and same with the command function name (`teamUsersArchiveCmdF`), test functions, etc.